### PR TITLE
Changed standard page title and team page custom title.

### PR DESCRIPTION
**Pull request by pmmcls**
## Description

Alter page title code to use Vonage brand and put the brand after the page title. Changed teams page title.

### CODE REVIEW

**Comment by mheap** [comment]
@fabianrbz You were making changes around `title` recently. Any overlap here?

File: app/helpers/application_helper.rb [hunk 1]
```diff
@@ -11,11 +11,11 @@ def theme
 
   def title
     if @product && @document_title
-      "Nexmo Developer | #{@product.titleize} > #{@document_title}"
+      "#{@product.titleize} > #{@document_title} | Vonage API Developer"
```
Comment (fabianrbz) [inline app/helpers/application_helper.rb:14]: @pmmcls @mheap yes, this branch might be outdated.
This method was removed and the logic now lives here
https://github.com/Nexmo/nexmo-developer/blob/master/app/presenters/page_title.rb#L1

Comment (pmmcls) [reply app/helpers/application_helper.rb:14]: Interesting - I only branched a couple of days ago. ok please reject the change then

Comment (pmmcls) [reply app/helpers/application_helper.rb:14]: is that on master? I've just done a pull and page_title presenter isn't there

Comment (pmmcls) [reply app/helpers/application_helper.rb:14]: Ok not to worry - I'm obviously not doing something right at this end

**Pull request closed without merging**